english/public-speaking/Toastmasters.md

```diff
@@ -0,0 +1,7 @@
+To enhance your public speaking and leadership skills, join a Toastmasters Club in your area.
+
+What is Toastmasters?
+
+Toastmasters International is a non-profit educational organization that teaches public speaking and leadership skills through a worldwide network of clubs. Headquartered in Englewood, Colo., the organization's membership exceeds 357,000 in more than 16,600 clubs in 143 countries. Since 1924, Toastmasters International has helped people from diverse backgrounds become more confident speakers, communicators, and leaders. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.